src/components/tooltip/Tooltip.tsx

```diff
@@ -222,6 +223,9 @@ class Tooltip extends React.Component<TooltipProps, State> {
             theme,
         } = this.props;
 
+        const childAriaLabel = getProp(children, 'props.aria-label');
+        const isLabelMatchingTooltipText = Boolean(childAriaLabel && childAriaLabel === text);
```

Comment:
Can we go with `const isLabelMatchingTooltipText = !!childAriaLabel && childAriaLabel === text;`?

Reply:
hmm I think... yes